modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -7,9 +7,33 @@
 
 The default installation of metering configures Hive to use an embedded Java database called Derby. This is unsuited for larger environments and can be replaced with either a MySQL or PostgreSQL database. Use the following example configuration files if your deployment requires a MySQL or PostgreSQL database for Hive.
 
-There are 4 configuration options you can use to control the database used by Hive metastore: url, driver, username, and password.
+There are three configuration options you can use to control the database used by Hive metastore: url, driver, and secretName.
 
-Use the example configuration file below to use a MySQL database for Hive:
+Create your MySQL or Postgres instance with a username and password. Then create a secret using the OpenShift CLI or a yaml file.. The `secretName` you create for this secret must map to the `spec.hive.spec.config.db.secretName` field in the MeteringConfig resource.
+
+To create a secret in Openshift CLI use the following command:
+
+[source,terminal]
+----
+$ oc --namespace openshift-metering create secret generic <YOUR_SECRETNAME> --from-literal=username=<YOUR_DATABASE_USERNAME> --from-literal=password=<YOUR_DATABASE_PASSWORD>
+----
+
+To create a secret using a yaml file, use the following example file:
```

Comment:
I'd include the command to apply the YAML to create the secret. Also, don't you still need to use `oc` to apply the secret so that the cluster can know about it? You might want to rethink the division of methods or just include one.